docs/conf.py

```diff
@@ -40,7 +48,24 @@
 # -- Options for HTML output ----------------------------------------------
 
 # We want to set the RTD theme, but not if we're on RTD.
-if os.environ.get('READTHEDOCS', '') != 'True':
+if os.environ.get('READTHEDOCS', '') == 'True':
```

Comment:
I meant `os.getenv('foo')` is equivalent to `os.environ.get('foo', None)` is equivalent to `os.environ.get('foo')`. Sending now.